docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -254,6 +254,27 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
 
 .. _datetime mapping:
 
+.. _oracle-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in
+Oracle. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/comment`
```

Comment:
These 3 (CREATE SCHEMA, DROP SCHEMA and COMMENTS) are not supported in Oracle connector.

Reply:
Updated here: https://github.com/trinodb/trino/pull/8844